docs/roadmap.md

```diff
@@ -24,27 +24,17 @@ Disclaimer:
 
 The following list of features are planned to be addressed in the short term, and incorporated in the next release:
 
--   "nosignal" actions supported in HA scenarios
--   Add astronomic clock support to timed rules (included sunrise and sunset functions for EPL statements)
--   Timezone support in EPL statements
--   Support of NGSI filter to support updating to groups of entities (commands)
--   Upgrade Esper library from 6.1.0 to 7.1.0
--   Full support for pagination in APIs /rules and /vrules
--   Explicit casting in EPL rules is no longer needed (full support of NGSI-v2 based typing)
--   Software quality improvement based on ISO25010 recommendations
--   Enhanced multi-tenancy support. Leverage different "Fiware-Service" and "Fiware-ServicePath" in rules
 -   Swagger documentation
+-   Upgrade Esper to 8.8
 
 ## Medium term
 
 The following list of features are planned to be addressed in the medium term, typically within the subsequent
 release(s) generated in the next **9 months** after next planned release:
 
--   Improve template processing, extending it to StructuredValue/JSON attributes.
--   Support for dealing with several Context Brokers from the same Perseo instance. To do so, update actions will
-    support to configure target context broker, if not provided such configuration, the default context broker will be
-    used.
 -   Rule scope beyond subservice: support dynamic creation of EPL context associated to notification path
+-   Accept scripts and java snipets
+-   No signal rules based in NGSIv2 API instead of MongoDB direct access #549
```

Comment:
PR https://github.com/telefonicaid/perseo-fe/pull/695 is solving this issue?

Reply:
No. Issue #549 is still pending.

Reply:
NTC